src/PhpCs/FiveLab/Sniffs/Strings/CyrillicSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the FiveLab CiRules package
+ *
+ * (c) FiveLab
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types = 1);
+
+namespace FiveLab\Component\CiRules\PhpCs\FiveLab\Sniffs\Strings;
+
+use FiveLab\Component\CiRules\PhpCs\FiveLab\ErrorCodes;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class CyrillicSniff implements Sniff
```

Comment:
This sniff should check only ASCII codes, without any cyrillic map, because must only support ASCII printable characters (character code 32-127).

https://www.ascii-code.com/